Makefile

```diff
@@ -1,6 +1,9 @@
 
 # Image URL to use all building/pushing image targets
 IMG ?= controller:latest
+SIDECAR_IMG ?= quay.io/csiaddons/k8s-sidecar
+TAG?= canary
```

Comment:
use latest instead of canary?

Reply:
> use latest instead of canary?

During image pull from registry,  using `latest` will usually point to the latest released version, that's why I picked `canary`.
Let me know if this sounds like sufficient reason?

Reply:
yes correct as we are not tagging the latest released with the latest tag we can use it for master/main builds so if anyone is using it will always get the latest build from main.